### ISSUE TITLE
Speaking range does not account for widescreen

### ISSUE THREAD

**Issue by Mickyan** [tgstation/tgstation#52827]
The speaking range no longer matches the visual range causing situations where someone can be visible but if they're two tiles near the left-right borders of the screen you will not hear their messages.

This is very confusing especially for new players as there's no visual feedback whatsoever for this and is very unintuitive as you can still examine with shift clicking but won't see dialog nor visible messages.

![speak](https://user-images.githubusercontent.com/38563876/89743271-c650df00-daa1-11ea-8594-c7fbb3ffa859.png)

**Comment by LemonInTheDark** [comment]
there's a lot of stuff like this, really a decision needs to be made about what the "intended" range is and what is an unfair advantage

**Comment by Mickyan** [comment]
I would imagine widescreen considering it's been purposefully made the default

**Comment by Jared-Fogle** [comment]
Yeah, we don't need to be balancing around library computer monitors.

**Comment by LemonInTheDark** [comment]
It should really be decided by a global var set to the max view range decided in config, but that's a lot of work and I'm lazy

**Comment by timothymtorres** [comment]
The problem with increasing speaking range to widescreen is that people would hear messages from people who aren't within view.  You shouldn't be getting a hearing advantage by using widescreen IMO.  This really shouldn't be considered a bug.

**Comment by Mothblocks** [comment]
What advantage? We can just use widescreen view for hearing distance all the time. The disadvantage then would be not being able to see who's talking, but you're on library mode and already can't see the edges of your screen. The actual number of people not using widescreen is so absurdly small (ping me if you want the numbers, I collected them a while ago) that we should not care so much about an advantage like that.

**Comment by LemonInTheDark** [comment]
the only issue would be hearing stuff from people "above" the top of your screen

**Comment by Mothblocks** [comment]
Surely we can check horizontal and vertical distance
